talhao/urls.py

```diff
@@ -1,11 +1,19 @@
+from core.consts.urls import UUID4_URL
+
 from django.conf.urls import url
 from django.urls import include
 
-from talhao.views.talhao import TalhaoAPIView
+from talhao.views.talhao import TalhaoAPIView, TalhaoDetailAPIView
 
 
 urlpatterns = [
     url(r'^talhao/', include([
-        url(r'^$', TalhaoAPIView.as_view(), name='talhao-create')
+        url(r'^$', TalhaoAPIView.as_view(), name='talhao-create'),
+
+        url(
+            r'^(?P<idTalhao>{})/historico/plantio/$'.format(UUID4_URL),
```

Comment:
```suggestion
            r'^(?P<idTalhao>{})/$'.format(UUID4_URL),
```
Se é um detail de talhão, com detalhamento de suas informações e plantios, não faz sentido ficar em uma rota /historico/plantios, pois não é retornado apenas os plantios do talhão. Seguir o proprio padrão no rest nesse caso é mais adequado

Reply:
Na realidade, o correto no detail do talhão é retornar somente o plantio que está ativo, porém esse histórico retornaria todos os plantios que já aconteceram, não temos a rota de detail porque já está sendo recebido tudo pela propriedade, se houver a necessidade dessa rota de detail do talhão eu poderia fazer, mas será que tem mesmo ?

Reply:
Na realidade, no detalhe de talhao deveria retornar todos os plantios, visto que é o detalhe dele. A rota /historico/plantios está agindo como o detail porque oq é retornado não é apenas plantios é todo o detalhe do talhao.

Reply:
Ok, seu comentário faz sentido, estarei aplicando a mudança necessária.

Reply:
Fixed